dev/create-release/release-build.sh

```diff
@@ -150,6 +150,7 @@ if [[ "$1" == "package" ]]; then
     NAME=$1
     FLAGS=$2
     ZINC_PORT=$3
+    BUILD_PIP_PACKAGE=$4
```

Comment:
This style of positional boolean flag won't really work if you ever need to have more than one of these arguments. That said, this is fine for now and if we're really going to add any more complexity here then we should probably just rewrite this whole script in Python.